roles/openshift_certificate_expiry/tasks/main.yml

```diff
@@ -1,4 +1,9 @@
 ---
+- name: Ensure python dateutil library is present
+  yum:
+    name: python-dateutil
```

Comment:
Need to verify the package name for fedora, python3-dateutil.  Example: https://github.com/openshift/openshift-ansible/blob/master/playbooks/init/base_packages.yml#L41

Reply:
Done.